### DISCUSSION TITLE
Multi-user / Team Workspaces?

### DISCUSSION THREAD

**Discussion by mikeckennedy** [anyproto/contributors#170] [Anytype apps]
Hi folks,

First of all, thanks for the awesome app. I'm really enjoying using it and hope to have something great to share in the show & tell section someday.

Part of that *might* eventually involve multiple people working on a project. Do you have any plans to support team features (shared pages or probably best shared workspaces)?

I have a couple of projects that I could put in there that would be excellent for multiple people to collaborate on but it also will inform what I put in for future projects and what I manage outside in Trello or Github projects.

Thank you!

**Comment by fuksman** [comment]
Hi @mikeckennedy! 

Thank you for your kind words :)

We aim to release multiplayer (aka shared spaces) in the end of April.

**Reply by mikeckennedy** [reply]
That's great, thank you @fuksman
